src/frontend/mame/infoxml.cpp

```diff
@@ -220,6 +220,12 @@ constexpr char f_dtd_string[] =
 		"\t\t\t<!ATTLIST driver emulation (good|imperfect|preliminary) #REQUIRED>\n"
 		"\t\t\t<!ATTLIST driver cocktail (good|imperfect|preliminary) #IMPLIED>\n"
 		"\t\t\t<!ATTLIST driver savestate (supported|unsupported) #REQUIRED>\n"
+		"\t\t\t<!ATTLIST driver type (arcade|console|computer|other) #REQUIRED>\n"
+		"\t\t\t<!ATTLIST driver requiresartwork (yes|no) #REQUIRED>\n"
+		"\t\t\t<!ATTLIST driver clickableartwork (yes|no) #REQUIRED>\n"
+		"\t\t\t<!ATTLIST driver unofficial (yes|no) #REQUIRED>\n"
+		"\t\t\t<!ATTLIST driver nosoundhardware (yes|no) #REQUIRED>\n"
+		"\t\t\t<!ATTLIST driver incomplete (yes|no) #REQUIRED>\n"
```

Comment:
`clickableartwork` shouldn’t be here, and neither should `type`.  The others are probably better off having specified defaults rather than `#REQUIRED` and then being omitted where they don’t apply.